charts/invenio/templates/invenio-configmap.yaml

```diff
@@ -20,5 +20,8 @@ data:
   INVENIO_DATACITE_ENABLED: "False"
   INVENIO_LOGGING_CONSOLE_LEVEL: "WARNING"
   {{- range $key, $value := .Values.invenio.extra_config }}
-  {{ $key }}: {{ $value | toYaml | indent 4 | trim }}
+  {{ $key }}: {{ $value $ | toYaml | indent 5 | trim }}
```

Comment:
I think you missed removing the `$` here when you removed `tpl`

Reply:
Good catch!